content/blog/2024/09/25/2024-09-25-board-officer-election-announcement.adoc

```diff
@@ -0,0 +1,132 @@
+---
+
+**Voter registration has started for the 2024 Jenkins Governance Board and Officer elections!**
+
+== Voter registration
+
+Nominations for the 2024 Jenkins Governance Board and Officer elections have finished.
+Voter registration opened September 16, 2024 and will continue through October 31, 2024.
+
+Participation in the election process requires registering an account on the link:https://community.jenkins.io[Jenkins community forums] and at least one contribution made before September 1, 2024.
+When registering, you can use an existing GitHub account or create a new account specifically for link:https://community.jenkins.io[Jenkins community forums].
+
+image:/images/post-images/jenkins-is-the-way/register-button.png[link="https://community.jenkins.io/g/election-voter-2024", role=center, height=48]
+
+All community members who meet the requirements and are interested in voting should join the link:https://community.jenkins.io/g/election-voter-2024[election-voter-2024] group during the registration period.
+Previous elections utilized their own groups, so joining the link:https://community.jenkins.io/g/election-voter-2024[election-voter-2024] group is required for participation.
+
+Refer to the link:/project/election-walkthrough/#join-the-annual-election-group[voter registration tutorial] for detailed registration steps.
+
+We have made the decision to trust participants and not require that you provide evidence of contribution as part of your voter registration.
+We reserve the right to ban any account from the election process if we identify abuse.
+Once registration is over, a list of email addresses will be sent to the link:https://civs1.civs.us/[Condorcet Internet Voting Service (CIVS)].
+
+== Nominees
+
+We will be electing three members of the Jenkins Governance Board from among six candidates.
+The six candidates for the Jenkins Governance Board are:
+
+* Alex Earl - link:https://contributors.jenkins.io/pages/contributors/alex-earl/[contributor spotlight] and link:/blog/authors/slide_o_mix/[author biography]
+* Alexander Brandes - link:https://contributors.jenkins.io/pages/contributors/alexander-brandes/[contributor spotlight] and link:/blog/authors/notmyfault/[author biography]
+* Kris Stern - link:https://contributors.jenkins.io/pages/contributors/kris-stern/[contributor spotlight] and link:/blog/authors/krisstern/[author biography]
+* Oleg Nenashev - link:/blog/authors/oleg_nenashev/[author biography]
+* Stefan Spieker - link:https://contributors.jenkins.io/pages/contributors/stefan-spieker/[contributor spotlight]
+* Valentin Delaye - link:https://contributors.jenkins.io/pages/contributors/valentin-delaye/[contributor spotlight]
+
+We will be electing the Jenkins Release Officer from among two candidates.
+The two candidates for Jenkins Release Officer are:
+
+* Alex Earl - link:https://contributors.jenkins.io/pages/contributors/alex-earl/[contributor spotlight] and link:/blog/authors/slide_o_mix/[author biography]
```

Comment:
Quick note completely unrelated to this PR: having the contributor spotlight added to the author page on jenkins.io would be nice!

Reply:
It would also avoid giving two links per person if there is a contributor spotlight.